src/components/account/LoginCard.tsx

```diff
@@ -33,21 +33,26 @@ interface Props {
 
 export default function LoginCard(props: Props): ReactElement {
   const history = useHistory();
+  const [isDisabled, setDisabled] = useState(false);
```

Comment:
This is to process the login request and not allow the user to press login multiple times right?

Reply:
yeah exactly, same for register card, we don't want to allow them to spam it n times and make n requests to BE